README.md

```diff
@@ -332,97 +342,11 @@ As of release 1.2.0, the following are not implemented and will be covered in fu
-
-```
-git clone git@github.com:ably/ably-go.git --recurse-submodules
-```
- 
-The tests can be run with the commands: 
- 
-```
-export ABLY_PROTOCOL="application/json" && go test -tags=integration -p 1 -race -v -timeout 120m ./...
-export ABLY_PROTOCOL="application/x-msgpack" && go test -tags=integration -p 1 -race -v -timeout 120m ./...
-``` 
-
-Depending on which protocol they are to be run for. It is also necessary to clean the test cache in between runs of these tests which can be done with the command: 
-
-```
-go clean -testcache
-``` 
-
-When adding new integration tests, the following build tag must be included at the top of the file to exclude these tests for running in CI as part of the Unit test step.
-
-```
-//go:build !unit
-// +build !unit
-```
-
-## Release process
-
-Starting with release 1.2, this library uses [semantic versioning](http://semver.org/). For each release, the following needs to be done:
-
-1. Create a branch for the release, named like `release/1.2.3` (where `1.2.3` is the new version number)
-2. Replace all references of the current version number with the new version number and commit the changes
-3. Run [`github_changelog_generator`](https://github.com/github-changelog-generator/github-changelog-generator) to automate the update of the [CHANGELOG](./CHANGELOG.md). This may require some manual intervention, both in terms of how the command is run and how the change log file is modified. Your mileage may vary:
-  - The command you will need to run will look something like this: `github_changelog_generator -u ably -p ably-go --since-tag v1.2.3 --output delta.md`
-  - Using the command above, `--output delta.md` writes changes made after `--since-tag` to a new file
-  - The contents of that new file (`delta.md`) then need to be manually inserted at the top of the `CHANGELOG.md`, changing the "Unreleased" heading and linking with the current version numbers
-  - Also ensure that the "Full Changelog" link points to the new version tag instead of the `HEAD`
-4. Commit this change: `git add CHANGELOG.md && git commit -m "Update change log."`
-5. Make a PR against `main`
-6. Once the PR is approved, merge it into `main`
-7. Add a tag to the new `main` head commit and push to origin such as `git tag v1.2.3 && git push origin v1.2.3`
-8. Create the release on Github, from the new tag, including populating the release notes
-9. Create the entry on the [Ably Changelog](https://changelog.ably.com/) (via [headwayapp](https://headwayapp.co/))
-
-## Further information
-
-Broaden your knowledge of realtime in Go with these useful materials:
-
-* [Building realtime apps with Go and WebSockets: client-side considerations](https://ably.com/topic/websockets-golang)
-* [Guide to Pub/Sub in Golang](https://ably.com/blog/pubsub-golang)
-
 ## Support and feedback
```

Comment:
This has a slightly different name in the template https://github.com/ably/ably-common/blob/main/templates/sdk-readme.md#support-feedback-and-troubleshooting

```suggestion
## Support, feedback and troubleshooting
```